lib/round.rb

```diff
@@ -0,0 +1,32 @@
+require 'pry'
+class Round
+  attr_reader :deck, :turns, :current_card
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @current_card = deck.cards[0]
+  end
+
+  def take_turn(guess)
+    new_turn = Turn.new(guess, current_card)
+    add_turn_to_turns(new_turn)
+#    move_to_next_card
+    new_turn
+  end
+
+  def add_turn_to_turns(new_turn)
+    @turns.push(new_turn)
+  end
+
+  def move_to_next_card
+    current_index = deck.cards.index(current_card)
+    next_index = current_index + 1
+    @current_card = deck.cards[next_index]
+  end
+
+  def correct?
+    @guess == @card.answer
```

Comment:
Again, make sure to be cautious when using the `@` sign. For instance, if you made a typo here and said `@guess = @card.answer`, you have now set `@guess` to the card answer for everything this class uses. If you had done `guess = card.answer` instead, the bug would be contained within the one method. This makes it easier to trace the error.